src/general/DaylightProperties.cs

```diff
@@ -0,0 +1,17 @@
+﻿public class DaylightProperties
+{
+    public float Maximum;
```

Comment:
I can't tell which values here are allowed to change, it would be super awesome to make these properties with setters only for the values that are allowed to be changed.